akka-docs/src/main/paradox/routing.md

```diff
@@ -608,13 +608,13 @@ insight into how consistent hashing is implemented.
 
 There is 3 ways to define what data to use for the consistent hash key.
 
- * You can define @scala[`hashMapping`]@java[`withHashMapper`] of the router to map incoming
+ * You can define @scala[`hashMapping`]@java[@javadoc[withHashMapper](akka.routing.ConsistentHashingRoutingLogic#withHashMapper(akka.routing.ConsistentHashingRouter.ConsistentHashMapper))] of the router to map incoming
```

Comment:
Should I add some links for Scala documentation?

Reply:
I think you can link to https://doc.akka.io/api/akka/current/akka/routing/ConsistentHashingPool.html#hashMapping:akka.routing.ConsistentHashingRouter.ConsistentHashMapping - but that can be a follow-up pr!